app/jobs/titre_identite_watermark_job.rb

```diff
@@ -1,9 +1,24 @@
 class TitreIdentiteWatermarkJob < ApplicationJob
+  class FileNotScannedYetException < StandardError
```

Comment:
```suggestion
  class FileNotScannedYetError < StandardError
```

A moins que tu veuilles délibérément faire une subtile distinction entre une erreur et et qqchose d'exceptionnel ?

Reply:
Non, je suis chaud pour `Error`, alley.